ruleset/sca/sles/15/cis_sles15_linux.yml

```diff
@@ -2314,8 +2279,12 @@ checks:
       - 'f:/etc/audit/auditd.conf -> r:^\s*action_mail_acct\s*=\s*root'
       - 'f:/etc/audit/auditd.conf -> r:^\s*admin_space_left_action\s*=\s*halt'
 
-  - id: 7599
-    title: Ensure events that modify user/group information are collected
+# 4.1.2.4 Ensure audit_backlog_limit is sufficient - Not implemented
+# 4.1.3 Ensure events that modify date and time information are collected - Not implemented
+
+# 4.1.4 Ensure events that modify user/group information are collected
+  - id: 7584
```

Comment:
Rules with pipes in this check have to be wrapped into `sh -c`.

Same for checks 7585-7589.